.travis.d/gen.rb

```diff
@@ -89,6 +100,29 @@ def load_app_data(data, app_config, dome_name, app_name)
   org_name = git_parts[-2]
   slog = "#{org_name}/#{repo_name}"
 
+  # filestuff
+  files = {}
+  if app_config['files'].is_a? Hash
```

Comment:
error if its not a Hash, if it's not a hash then they are using it wrong and people will get what they don't expect